request-manager/resources/migrations/v003_add_id_to_timestamp_indexes.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `requests`
+  DROP INDEX `created_at`,
+  ADD INDEX (`created_at`, `request_id`);
```

Comment:
I think we should also drop the `state` index and replace it with (`state`, `created_at`, `request_id`). That will get used in the list view UI which shows recent requests grouped by state.

Reply:
You're right, we'll probably be filtering by state a lot.

Reply:
Actually, there's also no point in dropping and recreating the `created_at` index — `request_id` is the primary key so it's already in the index.